README.rst

```diff
@@ -0,0 +1,16 @@
+BAP IDA Python
+==============
+
+This package provides the necessary IDAPython scripts required for
+interoperatibility between BAP and IDA Pro.
+
+Installation
+------------
+
+All of the requisite scripts can be installed via ``pip`` using::
+
+    pip install git+git://github.com/BinaryAnalysisPlatform/bap-ida-python
+
+Now all that remains is to inform IDA about where the scripts are and how to access BAP. This process is simplified greatly using the plugin installer. The installer can be run using::
+
+    python -c 'from bap_ida_python.loader.installer import main; main()'
```

Comment:
It can be simplified down further if necessary. I spent quite some time to see if we could automate it through pip as a post-install, but believe me, this turns out to be more robust on more systems. By tagging it along as a post-install, it fails under certain scenarios, and works fine under others. Hence, I prefer the more robust (though more verbose) solution. Also, it is a single copy-paste.

Alternative is to install a `bap-ida-python-plugin-installer` or similarly named utility which I find to be overkill, since it is a one-shot thing.